server/ngwmn/tests/services/test_ngwmn.py

```diff
@@ -21,7 +180,7 @@ def setUp(self):
         self.test_xml = '<site><agency>DOOP</agency><id>BP-1729</id></site>'
 
     @mock.patch('ngwmn.services.ngwmn.r.get')
-    def test_success(self, r_mock):
+    def test_get_iddata__success(self, r_mock):
```

Comment:
These renames look appropriate given the tests... would it also make sense to remove "lithology" from the class name? (TestGetWellLithography)

Reply:
I suppose. BTW, I renamed them originally because I thought I was going to add methods to the test class for statistics. Later (probably days later) I noticed that there was more than one test class in the file. I had forgotten that I changed he names.